Cargo.toml

```diff
@@ -133,6 +133,10 @@ codspeed-criterion-compat = { version = "2.3.3", default-features = false }
 glob                      = { version = "0.3.1" }
 mime_guess                = { version = "2.0.4" }
 language-tags             = { version = "0.3.2" }
+# tsify                     = { version = "0.4.5" }
+tsify                     = { git = "https://github.com/siefkenj/tsify.git", rev = "36f42be" }
```

Comment:
What's missing from tsify? Do we need to publish another crate fork this? We don't want a git dependency for many reasons ...

Reply:
Yep, fortunately that fork only addresses ident != name (as in, the name of the serde entity does not match the ident in rust), which is used in exactly two places - TSMappedTypeModifierOperator and TSTypeOperator.

I'll remove the original serde renames, that avoids the mismatching references (and the renames weren't that drastic anyway, I can't imagine all that many crate consumers breaking because of this), and the need for the fork.